ros_ign_image/CMakeLists.txt

```diff
@@ -16,11 +16,22 @@ find_package(catkin REQUIRED COMPONENTS
   roscpp
```

Comment:
Note that `ros_ign_image` is currently [being ignored](https://github.com/osrf/ros_ign/blob/dashing/ros_ign_image/COLCON_IGNORE) for ROS 2. But I think it doesn't hurt to update the cmake to match melodic's.